src/span.js

```diff
@@ -195,6 +216,20 @@ export default class Span {
         }
```

Comment:
I would rather do that in another diff, because the ability to use "setTags" is blocking my new sampler tags diff.